### PR TITLE
Displaying admin notification before login too.

### PR DESCRIPTION
**Pull request by zaparka**

### CODE REVIEW

File: app/controllers/notifications_controller.rb [hunk 1]
```diff
@@ -19,7 +19,7 @@ def update
   private
 
   def notification
-    @notification = Notification.first
+    @notification = Notification.first || Notification.create
```
Comment (leoapost) [inline app/controllers/notifications_controller.rb]: How about we use the `.first_or_create` method instead?

File: spec/features/admin_can_manage_the_message_info_spec.rb [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    end
+
+    scenario 'can view the message' do
+      visit '/'
+      expect(page).not_to have_content 'Edit banner'
+      expect(page).to have_content 'This is message from admin, hear, hear.'
+    end
+  end
+
+  context 'befor log in' do
```
Comment (leoapost) [inline spec/features/admin_can_manage_the_message_info_spec.rb]: `befor` is misspelled

Comment (zaparka) [reply spec/features/admin_can_manage_the_message_info_spec.rb]: @leoapost fixed :)

File: app/controllers/notifications_controller.rb (hunk 1 above)
Comment (zaparka) [reply app/controllers/notifications_controller.rb]: yes, cool. I didn't know about that one. I only knew about find_or_create :)

**Pull request merged**